released/packages/coq-unicoq/coq-unicoq.1.3+8.10/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+maintainer: "matthieu.sozeau@inria.fr"
+authors: [ "Matthieu Sozeau <matthieu.sozeau@inria.fr>" "Beta Ziliani <beta@mpi-sws.org>" ]
+dev-repo: "git+https://github.com/unicoq/unicoq.git"
+homepage: "https://github.com/unicoq/unicoq"
+bug-reports: "https://github.com/unicoq/unicoq/issues"
+license: "MIT"
+build: [
+  ["coq_makefile" "-f" "Make" "-o" "Makefile"]
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml"
+  "coq" {>= "8.10.0" & < "8.11~"}
+]
+synopsis: "An enhanced unification algorithm for Coq"
+url {
+  src: "https://github.com/unicoq/unicoq/archive/v1.3-8.10.tar.gz"
+  checksum: "md5=d41d8cd98f00b204e9800998ecf8427e"
```

Comment:
```suggestion
  checksum: "md5=cad6759caf66fe33967fa06c79caf223"
```

Reply:
Ah, I noticed the failure and fixed it too. Thanks @clarus and sorry for the mess!